sass/true/_results.scss

```diff
@@ -31,6 +34,11 @@
     $message: _true-report-message('global', 'single');
     @include _true-message($message, 'debug');
   }
+
+  @if $stats {
```

Comment:
Okay I added the stats to the summary.

About the `$true-terminal-output` setting: There may be some context I am missing since the issue was last discussed months ago. In one of the comments you mentioned that getting these stats would be something that can be turn on or off. I don't know if the `$true-terminal-output` setting existed then, but now that we're discussing it I am not really seeing a point of having a `$stats` parameter to the mixin. I think just using the value `$true-terminal-output` would be fine (I think this is what you're suggesting). What do you think?